eng/Versions.props

```diff
@@ -75,6 +75,10 @@
     <MicrosoftExtensionsDiagnosticsHealthChecksVersion>9.0.2</MicrosoftExtensionsDiagnosticsHealthChecksVersion>
     <MicrosoftExtensionsHttpPollyVersion>9.0.2</MicrosoftExtensionsHttpPollyVersion>
     <MicrosoftExtensionsObjectPoolVersion>9.0.2</MicrosoftExtensionsObjectPoolVersion>
+    <!-- Dependencies from https://github.com/dotnet/efcore -->
+    <MicrosoftEntityFrameworkCoreSqliteVersion>9.0.2</MicrosoftEntityFrameworkCoreSqliteVersion>
+    <!-- Dependencies from https://github.com/dotnet/arcade -->
+    <MicrosoftDotNetBuildTasksTemplatingVersion>9.0.0-beta.25111.5</MicrosoftDotNetBuildTasksTemplatingVersion>
```

Comment:
Note that these two dependencies are the only ones that will be automatically updated. Third party dependencies are put lower in this file (in the "Manual" group) and will need to be manually updated. And, of course, any template dependencies defined in the extensions repo (such as `Microsoft.Extensions.AI`) inherit the repo-defined version and therefore implicitly stay up to date.

Reply:
I just realised that this won't work as you expect. For these packages to be updated via the automated dependency flow there must be an entry in Version.Details.xml. 
The maestro bot is querying Version.Details.xml to determine what dependencies must be updated.